frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebar.scss

```diff
@@ -5,6 +5,7 @@
   }
 
   &__header, &__content {
+    height: 33px; // fixed height header to avoid actions from bouncing the header
```

Comment:
eww :(

Reply:
Not sure why the `ActionsMenu` remounts... the useEffect hook it uses sets the state immediately for my use case and somehow manages to get a render in there and causes the UI to flicker the dropdown in.